CHANGES.md

```diff
@@ -34,11 +38,14 @@ Contributions by Matthew Davidson and Stefan van den Oord.
 
 * Fix backwards-compatibility for transport options
 * Bump Netty to 4.1.89.Final, and io_uring to 0.0.18.Final
-* Add `wrap-validation` middleware to validate Ring maps 
```

Comment:
Why is this line removed? A changelog should be conceptually append-only, no? Was it incorrect?